packages/angular/build/BUILD.bazel

```diff
@@ -28,6 +27,18 @@ ts_json_schema(
     src = "src/builders/ng-packagr/schema.json",
 )
 
+JSON_FILES = glob(
+    include = [
+        "src/**/schema.json",
+        "src/**/*.js",
```

Comment:
The `JSON_FILES` here is misleading since the glob is for a number of file types.

Reply:
:+1: I've named the other constant also `ASSET_FILES`. Will do the same here